it/lessons/basics/collections.md

```diff
@@ -101,7 +101,7 @@ iex> File.read("path/to/unknown/file")
 
 ## Liste di Keywords
 
-Le liste di keywords e le mappe sono collezioni associative di Elixir; entrambe implementano il modulo `Dict`. In Elixir, una lista di keywords è una speciale lista di tuple che hanno un atom come primo elemento; condividono le performance con le liste:
+Le liste di keywords (_keyword lists_) e le mappe sono collezioni associative di Elixir; entrambe implementano il modulo `Dict`. In Elixir, una lista di keywords è una speciale lista di tuple che hanno un atom come primo elemento; hanno le stesse performance delle liste:
```

Comment:
"Le liste di keywords" => "Le liste di parole chiave"
"associative di Elixir;" => "associative in Elixir:"
"In Elixir, una lista di keywords è una speciale lista di tuple che hanno un atom come primo elemento;" => "In Elixir, una lista di keywords è una speciale lista di tuple contenenti un atom come primo elemento."